osu.Game/Graphics/UserInterface/SearchTextBox.cs

```diff
@@ -33,6 +34,15 @@ public SearchTextBox()
             PlaceholderText = "type to search";
         }
 
+        protected override bool HandleAction(PlatformAction action)
+        {
+            // Shift-delete is unnecessary for search inputs, so its propagated up the input queue.
```

Comment:
> Shift-delete

on which platform does this combination map to shift-delete?

Answer: macOS (only, i believe)

> is unnecessary

why is it unnecessary

Answer: because we block arrow keys so you're never going to be "deleting", only "backspacing" (see `HandleLeftRightArrows`)

> for search inputs

not sure what a search input is, but probably not necessary to say this since we are inside the `SearchTextBox` class.

> so its propagated up the input queue.

i guess? but that's not what's happening from this component's perspective. all that's happening here is we are avoiding handling and blocking the action to allow for other game components to potentially consume the shortcut. also `its` -> `it's`

Want to try writing this one more time?